### DISCUSSION TITLE
new set up help

### DISCUSSION THREAD

**Discussion by Podster2020** [3DCoded/3MS#12] [Configuration Help]
![image](https://github.com/user-attachments/assets/32779251-03c5-460d-b9ab-241eeebc9868)

my set up so far, 4 extruders and looking to sort the 4 in to 1 sorted i have the 8 in to 1 i found which has sensors on each, but is this required?

using the filamenalist to rewind the filament, with the filament sensor on

willl i need a filament cutter? i have a sherpa mini there is a few options for the filament cutter for these

will i need a filament sensor for above the extruder or between the extruder and hotend?

sorry for so many questions, so much to decide on lol

**Comment by 3DCoded** [comment]
> which has sensors on each, but is this required?

No. The 3MS only requires a filament sensor between the splitter and the main extruder.

> willl i need a filament cutter?

No. The 3MS doesn't support a filament cutter...yet

> will i need a filament sensor for above the extruder or between the extruder and hotend?

Yes. You will need a filament sensor right above the main extruder

> sorry for so many questions, so much to decide on lol

No problem. Modular systems have so many options!
